packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -27,16 +27,18 @@ const double _kLeadingWidth = kToolbarHeight; // So the leading button is square
 
 // Bottom justify the kToolbarHeight child which may overflow the top.
```

Comment:
This comment will need to be updated

Reply:
Sorry, I did not understand.

Reply:
The child isn't "kToolbarHeight" any more